cmd_set.go

```diff
@@ -631,13 +633,18 @@ func (m *Miniredis) cmdSscan(c *server.Peer, cmd string, args []string) {
 				c.WriteError(msgSyntaxError)
 				return
 			}
-			_, err := strconv.Atoi(args[1])
+			count, err := strconv.Atoi(args[1])
 			if err != nil {
 				setDirty(c)
 				c.WriteError(msgInvalidInt)
 				return
 			}
-			// We do nothing with count.
+			if count < 0 {
```

Comment:
you can combine this branch and 637 into one condition `if err != nil || count < 0`